app/components/reaction_picker/index.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
+
+import ReactionPicker from './reaction_picker';
+
+function makeMapStateToProps() {
+    return function mapStateToProps(state) {
+        return {
+            theme: getTheme(state),
+            recentEmojis: state.views.recentEmojis,
+        };
+    };
+}
+
+export default connect(makeMapStateToProps)(ReactionPicker);
```

Comment:
```suggestion
function mapStateToProps(state) {
    return {
        theme: getTheme(state),
        recentEmojis: state.views.recentEmojis,
    };
}

export default connect(mapStateToProps)(ReactionPicker);
```

you don't really need a factory here